jetty-io/src/main/java/org/eclipse/jetty/io/ssl/SslConnection.java

```diff
@@ -104,7 +105,10 @@
         WAIT_FOR_FILL // Waiting for a fill to happen
     }
 
+    private final AtomicReference<HandshakeState> _handshake = new AtomicReference<>(HandshakeState.INITIAL);
     private final List<SslHandshakeListener> handshakeListeners = new ArrayList<>();
+    private final AtomicLong _bytesIn = new AtomicLong();
```

Comment:
We have been changing to LongAdder in other stats classes, so perhaps we should use here as well?

Reply:
I think `LongAdder` is more suited when it's contended, but here is not so I went for `AtomicLong`.